src/Tar.jl

```diff
@@ -11,6 +11,7 @@ end
 include("header.jl")
 include("create.jl")
 include("extract.jl")
+include("utils.jl")
```

Comment:
Do we really want to load the utils file just for the one function?

Reply:
I figured it was an oversight that it wasn't included.  Should we delete the other stuff in there, or is it useful to have around but not `include()`ed?

Reply:
It's useful for debugging: it lets you read a 512-byte header block and then parse out the individual fields.